terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -324,17 +324,6 @@ module "glue_s3_file_transfer_job" {
   ]
 }
 
-resource "aws_glue_trigger" "glue_s3_file_transfer_job_trigger" {
-  count    = local.enable_s3_file_transfer_trigger ? 1 : 0
-  name     = "${module.glue_s3_file_transfer_job.name}-trigger"
-  schedule = local.scheduled_s3_file_transfer_schedule
-  type     = "SCHEDULED"
-
-  actions {
-    job_name = module.glue_s3_file_transfer_job.name
-  }
-}
-
```

Comment:
This trigger has been moved to the ingestion jobs and modified to have a trigger per domain for isolation. This allows pausing the archiving job for a particular domain while it is being replayed.